mne/io/meas_info.py

```diff
@@ -51,6 +51,16 @@ class Info(dict):
     """ Info class to nicely represent info dicts
     """
 
+    def copy(self):
+        """Copy the instance
+
+        Returns
+        -------
+        info : instance of Info
+            The copied info.
+        """
+        return Info(super(Info, self).copy())
```

Comment:
mhmm. wondering where we use this. It will be a shallow copy, wondering if we really wan this,